### ISSUE TITLE
Package Repository Information

### ISSUE THREAD

**Issue by irmerk** [accordproject/techdocs#341]
# Feature Request 🛍️

Suggested by @steelbrain, add repository information to package.json, see [npm's documentation](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#repository)

## Use Case

It'll point people discovering you packages on NPM to you Github repository

## Required Repositories

- [x] [`ergo`](https://github.com/accordproject/ergo)
- [x] [`cicero`](https://github.com/accordproject/cicero)
- [x] [`concerto`](https://github.com/accordproject/concerto)
- [x] [`markdown-transform`](https://github.com/accordproject/markdown-transform)
- [x] [`web-components`](https://github.com/accordproject/web-components)

**Comment by Aniruddha-Shriwant** [comment]
Hello @irmerk I'm interested in this 🙂

**Comment by Aniruddha-Shriwant** [comment]
I had seen this repositories:
> *  [`ergo`](https://github.com/accordproject/ergo)
> *  [`cicero`](https://github.com/accordproject/cicero)
> *  [`concerto`](https://github.com/accordproject/concerto)
> *  [`markdown-transform`](https://github.com/accordproject/markdown-transform)
> *  [`web-components`](https://github.com/accordproject/web-components)

and found that these repositories ( [`ergo`](https://github.com/accordproject/ergo),  [`cicero`](https://github.com/accordproject/cicero), [`markdown-transform`](https://github.com/accordproject/markdown-transform)) had their repository information to `package.json`

So I will make PR in  [`concerto`](https://github.com/accordproject/concerto) & [`web-components`](https://github.com/accordproject/web-components) soon.

**Comment by steelbrain** [comment]
Given that the linked repositories contain mono-repos, I think the `directory` attribute is missing

![image](https://user-images.githubusercontent.com/4278113/107680569-d56a5100-6cbf-11eb-8f61-7bcdc20c9aac.png)

Should this be re-opened?

**Comment by Aniruddha-Shriwant** [comment]
May I work on this?

**Comment by Aniruddha-Shriwant** [comment]
@irmerk, I think we can close this now...
